specification/dimensions/adjustmenttype.md

```diff
@@ -0,0 +1,54 @@
+# adjustment type
+
+Adjustment Type indicates the record represents a change to your monthly invoice through after-the-fact adustements (e.g. refunds, pricing changes). An adjustment can be positive or negative.
+Adjustment Type can be used by practitioners to assess any impacts to spend from positive or negative adjustments.
+
+The AdjustmentType column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the allowed values.
+
+- Usage Correction
```

Comment:
Move all of these to the table below and add definitions.

Did you get these from somewhere? This is a lot more detailed than I was expecting 🙂 

We should add an "Other" option since we likely won't catch everything.

Reply:
I got these from invoices i have had from Google, AWS and Azure.

Reply:
Can you add the explicit things you saw and how they're tracked to the supporting content?